src/views/Home/Home.scss

```diff
@@ -15,3 +15,7 @@
     margin-bottom: var(--medium-spacer);
   }
 }
+
+a {
+  color: inherit;
+}
```

Comment:
Could we use class names instead of generic attributes? I understand why we have this here, but I don't see the `a` tag anywhere within `Home` index view. Let's give the `a` tags a classname within the `Footer` component and style the attributes there.

Reply:
That's understandable - no problem!